test/views/claimToAdjustPoa/PoaAdjustedViewSpec.scala

```diff
@@ -102,12 +108,18 @@ class PaymentsOnAccountAdjustedViewSpec extends TestSupport{
         document.getElementById("p3").text shouldBe
           msgs("claimToAdjustPoa.success.checkYour") + " " + msgs("claimToAdjustPoa.success.link", taxYear.startYear.toString, taxYear.endYear.toString) + " " +
             msgs("claimToAdjustPoa.success.afterLinkText")
-        document.getElementById("p3").getElementById("link").attr("href") shouldBe taxYearSummaryUrl(true)
+        document.getElementById("p3").getElementById("TYSLink").attr("href") shouldBe taxYearSummaryUrl(true)
       }
-      "render the final paragraph" in new Setup(isAgent = true) {
+      "render the final paragraph when no overdue charges" in new Setup(isAgent = true) {
         document.getElementById("p4").text shouldBe
           msgs("claimToAdjustPoa.success.check") + " " + msgs("claimToAdjustPoa.success.whatYouOwe") + " " + msgs("claimToAdjustPoa.success.forUpcomingCharges")
-        document.getElementById("p4").getElementById("link").attr("href") shouldBe whatYouOweUrl(true)
+        document.getElementById("p4").getElementById("WYOLink").attr("href") shouldBe whatYouOweUrl(true)
```

Comment:
Could we have a view spec test to specifically test that the section is missing when the condition is false